datetimepicker/src/main/java/com/taransit/datetimepicker/dialog/DateTimePickerDialog.kt

```diff
@@ -49,12 +74,19 @@ class DateTimePickerDialog : DialogFragment(),
         dateAdapter = DateAdapter(object : DateAdapter.OnDateClickListener {
             override fun onDateClicked(date: Calendar) {
                 val context = context ?: return
-                DatePickerDialog(context,
```

Comment:
Pokud opravdu potřebujeme zaoblené rohy pro DatePickerDialog, udělal bych z toho samostatnou třídu. Ale nemyslím si, že to vůbec potřebujeme.